Milestone: Hello-Microverse Project

malikhaiderkhan opened the pull request:
I updated the index.html and style.css file by adding class attribute in it and modify the README.md file according to the reviewers and also added LICENSE to the repository.

malikhaiderkhan committed:
Add html file

malikhaiderkhan committed:
Add css file

malikhaiderkhan commented:
It is a simple Microverse project using html and css.

ademibowale reviewed (changes requested):
## STATUS: Changes Required :recycle:

### Hi @malikhaiderkhan , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._
- [x] Please kindly modify your readme file to suit the miroverse readme template  [This Template](https://github.com/microverseinc/readme-template/blob/master/README.md)._  

- [x] I suggest you to pull out the readme from your main branch to the feature branch (html-css) and modify it to suit the project and its personal details. As currently in your feature branch all i can see in your README file is a text (Hello World). Please kindly work on tis
    - Please Note: while walking through your readme file from the main branch, i notice the MIT licence is not properly added to your project, Kindly work on it

- [x] As stated in the project requirement, I would kindly suggest you give your `h1` in your `HTML file` a class name so that the class name can be used to style the content in the CSS file

E.g
```HTML
<h1 class="header">Hello Microverse</h1>
```

- [x] In your `CSS file`, I would also suggest you use the class name you have given the `h1` in the `HTML` to style your content, this is the requirement of this project and this will also show your ability to follow any requirement given in a project.

E.g
```CSS
.header {
  color: lightgreen;
}
```

- [x] Lastly, I would also suggest you work on the pull request title to make it more descriptive and denote what you have in the project milestone and this shows some level of professionalism when working on your project. I would suggest something like this "Milestone: Hello-Microver Project"

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

malikhaiderkhan committed:
Modify html/css/readme files

malikhaiderkhan committed:
Update README.md

malikhaiderkhan committed:
Update README.md

malikhaiderkhan committed:
Update README.md

malikhaiderkhan committed:
Delete package-lock.json

malikhaiderkhan committed:
Delete package.json

malikhaiderkhan committed:
Merge branch 'main' into html-css

malikhaiderkhan committed:
Update README.md

malikhaiderkhan committed:
Update README.md

malikhaiderkhan committed:
Update README.md

Strangeal commented on README.md:
```diff
@@ -31,14 +38,25 @@
+> Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
- [x] As specified by the previous reviewer you need to customize the readme template based on the needs of your project. Kindly remove the `live demo` section as this project doesn’t have a live version so it will be misleading to have it in your `README`. Please remember to remove it from your `Table of content` as well.

Strangeal commented on README.md:
```diff
@@ -50,20 +68,22 @@ In order to run this project you need:
 ### Setup
 Clone this repository to your computer
```
As specified by the previous reviewer you need to customize the readme template based on the needs of your project. This section of the `README` should outline the steps a user can use to clone your repository into their local machine. Example:
```sh
  cd Hello-Microverse
  git clone git@github.com:malikhaiderkhan/Hello-world.git
```

Strangeal commented on README.md:
```diff
@@ -50,20 +68,22 @@ In order to run this project you need:
-### Usage
-To run the project, execute the "index.html" file in your browser
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Install
+Install this project with:
 
```
- [x] As specified by the previous reviewer you need to customize the readme template based on the needs of your project. Kindly add instructions on how users can install all dependencies of your project after cloning it into their local machine. Example: 
```
npm install
```

Strangeal commented on README.md:
```diff
@@ -50,20 +68,22 @@ In order to run this project you need:
 
-### Key Features <a name="key-features"></a>
+### Run tests
+To run tests, run the following command: Open index.html with lite server in vscode.
```
- [x]As specified by the previous reviewer you need to customize the readme template based on the needs of your project. Kindly add a step by step instructions on how users can run `linters` and check for errors after cloning your repository. This way they will not find themselves wanting trying to figure out how to run tests. Example:

### Run Tests
To track linter errors locally follow these steps:  

Download all the dependencies run:
```
npm install
```
Track HTML linter errors run:
```
npx hint .
```
Track CSS linter errors run:
```
npx stylelint "**/*.{css,scss}"
```

Strangeal commented on README.md:
```diff
@@ -50,20 +68,22 @@ In order to run this project you need:
 
-> An exercise to test linters!
+### Deployment
+Coming Soon
```
- [x]  As specified by the previous reviewer you need to customize the readme template based on the needs of your project. Kindly remove the `deployment` section as this project is not going to be deployed. You should include it in projects that have live demos. Please remember to remove it from your `Table of content` as well.

Strangeal reviewed (changes requested):
Hi @malikhaiderkhan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![Alt Text](https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif)

 ## Required Changes ♻️
- [x]  As specified by the previous reviewer kindly note that when creating a pull request, it's important to provide a descriptive summary that clearly outlines all the changes you have implemented in your branch. This will help reviewers understand the context of your work and provide feedback that is relevant and actionable.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

malikhaiderkhan committed:
Update README.md

Delete Live Demo, Deployment section from the file and modify Setup, Run Tests and Install sections in the file.

malikhaiderkhan commented:
@Strangeal please just check it can I fulfill your requirements or not?

malikhaiderkhan committed:
Update README.md

Working on Readme file to look more attractive.

malikhaiderkhan reviewed (commented):
I updated the index.html and style.css file by adding class attribute in it and modify the README.md file according to the reviewers and also added LICENSE file to the repository.

victorokoroji reviewed (approved):
Hi @malikhaiderkhan , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
